zmon_aws_agent/aws.py

```diff
@@ -236,7 +238,7 @@ def get_running_apps(region, existing_entities=None):
 
                 if 'ImageId' in i:
                     images.add(i['ImageId'])
-                    ins['image_id'] = i['ImageId']
+                    ins['image'] = {'id': i['ImageId']}
```

Comment:
I think it would be better to always add an ``image`` field. If ``ImageId`` doesn't exist then we can always have an empty ``{}``.